examples/dockerfile2llb/main.go

```diff
@@ -32,7 +31,6 @@ func xmain() error {
 	var opt buildOpt
 	flag.StringVar(&opt.target, "target", "", "target stage")
 	flag.StringVar(&opt.partialImageConfigFile, "partial-image-config-file", "", "Output partial image config as a JSON file")
-	flag.StringVar(&opt.partialMetadataFile, "partial-metadata-file", "", "Output partial metadata sa a JSON file")
```

Comment:
Is there a reason to keep either of these options?

Not sure where we use `partial-image-config-file` either.